tests/integration/component.go

```diff
@@ -190,15 +185,13 @@ func componentTests(args ...string) {
 			Expect(desiredCompListJSON).Should(MatchJSON(actualCompListJSON))
 			cmpAllList := helper.CmdShouldPass("odo", append(args, "list", "--all")...)
 			Expect(cmpAllList).To(ContainSubstring("cmp-git"))
-			helper.CmdShouldPass("odo", append(args, "delete", "cmp-git", "-f")...)
```

Comment:
If the test spec is responsible to create the component then the same component should be deleted as part of the test pec. Because just to make sure certain operation after component create should not affect the component delete.